FirebaseInstallations/Source/Library/Public/FirebaseInstallations/FIRInstallations.h

```diff
@@ -85,23 +85,24 @@ NS_SWIFT_NAME(Installations)
 - (void)installationIDWithCompletion:(FIRInstallationsIDHandler)completion;
 
 /**
- * Retrieves (locally if it exists or from the server) a valid authorization token. An existing
- * token may be invalidated or expired, so it is recommended to fetch the auth token before each
- * server request. The method does the same as `Installations.authTokenForcingRefresh(:,
+ * Retrieves (locally if it exists or from the server) a valid installation auth token. An existing
+ * token may be invalidated or expired, so it is recommended to fetch the installation auth token
+ * before each server request. The method does the same as `Installations.authTokenForcingRefresh(:,
  * completion:)` with forcing refresh `NO`.
  * @param completion A completion handler which is invoked when the operation completes. See
  * `InstallationsTokenHandler` for additional details.
  */
 - (void)authTokenWithCompletion:(FIRInstallationsTokenHandler)completion;
 
 /**
- * Retrieves (locally or from the server depending on `forceRefresh` value) a valid authorization
- * token. An existing token may be invalidated or expire, so it is recommended to fetch the auth
- * token before each server request. This method should be used with `forceRefresh == YES` when e.g.
- * a request with the previously fetched auth token failed with "Not Authorized" error.
- * @param forceRefresh If `YES` then the locally cached auth token will be ignored and a new one
- * will be requested from the server. If `NO`, then the locally cached auth token will be returned
- * if exists and has not expired yet.
+ * Retrieves (locally or from the server depending on `forceRefresh` value) a valid Installation
```

Comment:
Suggest lowercase for consistency with the other instances.

When we're referring to "Installations SDK" generally I think the product init-cap makes sense.